OutOfSchool/OutOfSchool.EmailSender/IEmailSender.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Threading.Tasks;
+
+namespace OutOfSchool.EmailSender
+{
+    public interface IEmailSender
+    {
+        Task SendAsync(Message message);
+
```

Comment:
this method is only called on class where it is implemented

Reply:
removed from interface